docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cancer_center/node--cgov-cancer-center--full.html.twig

```diff
@@ -1,82 +1,71 @@
-{{ attach_library('cgov_common/cancer_center') }}
+{{ attach_library('cgov_common/cancer-centers') }}
 {% import '@cgov_common/content_page_macros.twig' as contentPageMacros %}
 <!-- ********************************* BEGIN Page Content ********************************** -->
-<cancer-center>
-    <div class="resize-content">
-        {{ drupal_block('page_options') }}
-        <div id="nvcgSlProfilePanel" class="profile-panel">
-          <div class="inst-profile-image">
+<article>
+  <div class="resize-content">
+    {{ drupal_block('page_options') }}
+    <div id="nvcgSlProfilePanel" class="profile-panel">
+      <div class="slot-item only-SI">
+        <!-- start profile panel content -->
+        <div class="inst-profile-image">
           {{ content.field_image_promotional }}
-          </div>
-          <div class="profile-panel-content">
-            <h1>
-              {{ node.label }}
-            </h1>
-            {% if content.field_parent_institution or content.field_institution_type %}
+        </div>
+        <div class="profile-panel-content">
+          <h1>{{ node.label }}</h1>
+          {% if content.field_parent_institution or content.field_institution_type %}
             <div class="profile-title">
-            {% endif %}
-              {% if content.field_parent_institution %}
               {{ content.field_parent_institution }}
-              {% endif %}
               {% if content.field_institution_type %}
-              <br />
-              {{ content.field_institution_type }}
-              {% endif %}
+                <br/>{{ content.field_institution_type }}
+              {%- endif -%}
             </div>
-            {% if content.field_org_head_name %}
             <p>
```

Comment:
I think there should still be an "if" around this p tag - the org_head_name and org_head_title fields aren't required, so they might both be empty. I believe it's best practice to only output the HTML tags if we need them, so we only want to output the p tags if org_head_name has a value. If that isn't the case, then the p tag isn't needed.

Reply:
Done.